HsYAML.cabal

```diff
@@ -14,7 +14,7 @@ copyright:           2015-2018 Herbert Valerio Riedel
                    , 2007-2008 Oren Ben-Kiki
 category:            Text
 build-type:          Simple
-tested-with:         GHC==8.6.1, GHC==8.4.3, GHC==8.4.1, GHC==8.2.2, GHC==8.0.2, GHC==7.10.3, GHC==7.8.4, GHC==7.6.3, GHC==7.4.2
+tested-with:         GHC==8.10.1, GHC==8.6.5, GHC==8.4.4, GHC==8.2.2, GHC==8.0.2, GHC==7.10.3, GHC==7.8.4, GHC==7.6.3, GHC==7.4.2
```

Comment:
If testing with GHC 8.4.1 was particularly important, I can add it back!

Reply:
It was... see #1 or #6 -- there was a screw-up in GHC 8.4.1 releases where GHC would massively violate the versioning contract in a way that was very hard to workaround... that's the reason of this particular testing point...

Reply:
Thanks for the explanation! I re-added 8.4.1 to CI.